cmsswdata.spec

```diff
@@ -90,9 +90,35 @@ Requires: data-GeneratorInterface-ReggeGribovPartonMCInterface
 
 %install
 
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/cmsswdata.xml
+  <tool name="cmsswdata" version="%v">
+    <client>
+      <environment name="CMSSWDATA_BASE" default="%{cmsroot}/%{cmsplatf}/%{pkgcategory}"/>
+      <environment name="CMSSW_DATA_PATH" default="$CMSSWDATA_BASE"/>
+EOF_TOOLFILE
+
+cat << \EOF_TOOLFILE > %i/searchpath.xml
+    </client>
+    <runtime name="CMSSW_DATA_PATH" value="$CMSSWDATA_BASE" type="path"/>
+EOF_TOOLFILE
+
+for toolbase in `echo %pkgreqs | tr ' ' '\n' | grep 'cms/data-'` ; do
+  toolver=`basename $toolbase`
+  pack=`echo $toolbase | cut -d/ -f2 | sed 's|data-||;s|-|/|'`
+  echo "      <flags CMSSW_DATA_PACKAGE=\"$pack=$toolver\"/>" >> %i/etc/scram.d/cmsswdata.xml
+  echo "    <runtime name=\"CMSSW_SEARCH_PATH\" default=\"%{cmsroot}/%{cmsplatf}/$toolbase\" type=\"path\"/>" >> %i/searchpath.xml
+done
+
+cat %i/searchpath.xml >> %i/etc/scram.d/cmsswdata.xml
+echo "  </tool>"      >> %i/etc/scram.d/cmsswdata.xml
+rm -f %i/searchpath.xml
+
 %post
 echo "%{BaseTool}_ROOT='$CMS_INSTALL_PREFIX/%{pkgrel}'" > $RPM_INSTALL_PREFIX/%{pkgrel}/etc/profile.d/init.sh
 echo "set %{BaseTool}_ROOT='$CMS_INSTALL_PREFIX/%{pkgrel}'" > $RPM_INSTALL_PREFIX/%{pkgrel}/etc/profile.d/init.csh
+echo "%{BaseTool}_PKGREQUIRED='%pkgreqs'" >> $RPM_INSTALL_PREFIX/%{pkgrel}/etc/profile.d/init.sh
+echo "set %{BaseTool}_PKGREQUIRED='%pkgreqs'" >> $RPM_INSTALL_PREFIX/%{pkgrel}/etc/profile.d/init.csh
```

Comment:
@mrodozov once test are done then please remove these lines, I think these are not needed any more ... right?